src/PostgREST/MainTest.hs

```diff
@@ -8,7 +8,7 @@ import           PostgREST.Config                     (AppConfig (..),
                                                        readOptions)
```

Comment:
Actually I don't know what this file is. I deleted it locally and was able to build the binary and run the tests, so I'd say get rid of it.